lib/mozart_fetcher/component.ex

```diff
@@ -26,8 +26,15 @@ defmodule MozartFetcher.Component do
          config = %Config{endpoint: endpoint},
          {:ok, %HTTPoison.Response{status_code: 200, body: body}}
        ) do
-    metric(config.id, endpoint, 200)
-    %{index: component_index, id: config.id, status: 200, envelope: Envelope.build(body)}
+    case Jason.decode(body, keys: :atoms) do
+      {:ok, data} ->
+        metric(config.id, endpoint, 200)
+        %{index: component_index, id: config.id, status: 200, envelope: data}
+
+      {:error, _err} ->
```

Comment:
Previously we logged when this errored. Is this something we need or found useful before?

Reply:
The `metric` method called below is perhaps slightly misleading as it will also log for errors. Although the message will be slightly different it does mean it will still log the errors